governance-exercises/Executive Vote 02.md

```diff
@@ -0,0 +1,162 @@
+*Reasoning & authorizing party*: Governance Facilitator's [Forum post](http://forum.makerdao.com/t/avc-member-participation-rewards-q2-2023/21459)
+
+AVC Members are MKR holders who organize into Voter Committees for the purpose of participating in Governance. This is the very first distribution of AVC Member Participation Rewards, which are to be paid on a quarterly basis. The total Participation Rewards to be distributed to AVC Members for Q2 will be **134.1 MKR**. If this executive proposal passes, the Participation Rewards will be distributed as seen below:
+
+| AVC Member       | Address                                                      | Amount (MKR) |
+| ---------------- | ------------------------------------------------------------ | ------------ |
+| IamMeeoh         | [0x47f7A5d8D27f259582097E1eE59a07a816982AE9](https://etherscan.io/address/0x47f7a5d8d27f259582097e1ee59a07a816982ae9) | 14.90        |
+| ACRE DAOs        | [0xBF9226345F601150F64Ea4fEaAE7E40530763cbd](https://etherscan.io/address/0xBF9226345F601150F64Ea4fEaAE7E40530763cbd) | 14.90        |
+| Space Xponential | [0xFF8eEB643C5bfDf6A925f2a5F9aDC9198AF07b78](https://etherscan.io/address/0xFF8eEB643C5bfDf6A925f2a5F9aDC9198AF07b78) | 11.92        |
+| Res              | [0x8c5c8d76372954922400e4654AF7694e158AB784](https://etherscan.io/address/0x8c5c8d76372954922400e4654AF7694e158AB784) | 14.90        |
+| LDF              | [0xC322E8Ec33e9b0a34c7cD185C616087D9842ad50](https://etherscan.io/address/0xC322E8Ec33e9b0a34c7cD185C616087D9842ad50) | 11.92        |
+| opensky          | [0x8e67ee3bbeb1743dc63093af493f67c3c23c6f04](https://etherscan.io/address/0x8e67ee3bbeb1743dc63093af493f67c3c23c6f04) | 14.90        |
+| David Phelps     | [0xd56e3E325133EFEd6B1687C88571b8a91e517ab0](https://etherscan.io/address/0xd56e3E325133EFEd6B1687C88571b8a91e517ab0) | 8.94         |
+| seedlatam.eth    | [0x0087a081a9b430fd8f688c6ac5dd24421bfb060d](https://etherscan.io/address/0x0087a081a9b430fd8f688c6ac5dd24421bfb060d) | 11.92        |
+| StableLab        | [0xbDE65cf2352ed1Dde959f290E973d0fC5cEDFD08](https://etherscan.io/address/0xbDE65cf2352ed1Dde959f290E973d0fC5cEDFD08) | 14.9         |
+| flipsidegov      | [0x300901243d6CB2E74c10f8aB4cc89a39cC222a29](https://etherscan.io/address/0x300901243d6CB2E74c10f8aB4cc89a39cC222a29) | 14.9         |
+
+
+
+### DAO Resolution for Monetalis Clydesdale James Asset Trust 1 (JAT1) Arrangement Surplus Return 
+
+*Reasoning*: Monetalis' [Forum post](http://forum.makerdao.com/t/clydesdale-quarterly-return-of-surplus-fund/21291)\
+*Authorizing party*: [Stability Scope Facilitator](http://forum.makerdao.com/t/clydesdale-quarterly-return-of-surplus-fund/21291/8)
+
+As of Q2 2023, the Monetalis Clydesdale JAT1 Arrangement has a surplus of approximately $6.8M to return to the MakerDAO Surplus Buffer. Monetalis seeks approval via a DAO Resolution to execute this latest Surplus Return Process. If this executive proposal passes, the following actions will occur:
```

Comment:
No idea if this number is correct, but I know there has been debate on what the correct number is. Imo I would just say something like: 'has a significant surplus' or something. The exact amount isn't critically important for governance to know here, and given there is debate on what it should be, probably shouldn't implicitly take the position that 6.8M is the correct number.